executor/show_test.go

```diff
@@ -446,14 +446,14 @@ func (s *testSuite2) TestShowCreateTable(c *C) {
 	tk.MustExec("create table t1(a int,b int)")
 	tk.MustExec("drop view if exists v1")
 	tk.MustExec("create or replace definer=`root`@`127.0.0.1` view v1 as select * from t1")
```

Comment:
please add some test cases like 
``` sql
select avg(a), * from t1;
select a+b, * , a from t1;
```

Reply:
@XuHuaiyu These test should add to `TestCreateView` in `ddl_test.go`?

Reply:
yes

Reply:
done